src/util/path.cpp

```diff
@@ -1,12 +1,19 @@
+// POSIX headers
```

Comment:
Elsewhere we use guards like `#ifdef __linux__`. Here you have opted to make it something that is configured in the build system.
We should come up with one policy, and follow it.